package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@dashevo/wallet-lib",
-  "version": "7.15.0",
+  "version": "7.15.1",
```

Comment:
We usually separate bump and actual changes into different PRs, otherwise the [conventional-changelog-cli](https://github.com/conventional-changelog/conventional-changelog/tree/master/packages/conventional-changelog-cli) which we use to generate changelogs is not working properly.

Reply:
Seems to be fixed by @antouhou update of 0.16 branch :D